tests/test_vmec.py

```diff
@@ -874,6 +866,369 @@ def test_vmec_save_2(VMEC_save):
+    np.testing.assert_allclose(
+        vmec.variables["chi"][:], desc.variables["chi"][:], atol=3e-5, rtol=1e-3
+    )
+    np.testing.assert_allclose(
+        vmec.variables["chipf"][:], desc.variables["chipf"][:], atol=3e-5, rtol=1e-3
+    )
+    np.testing.assert_allclose(
+        vmec.variables["Rmajor_p"][:], desc.variables["Rmajor_p"][:]
+    )
+    np.testing.assert_allclose(
+        vmec.variables["Aminor_p"][:], desc.variables["Aminor_p"][:]
+    )
+    np.testing.assert_allclose(vmec.variables["aspect"][:], desc.variables["aspect"][:])
+    np.testing.assert_allclose(
+        vmec.variables["volume_p"][:], desc.variables["volume_p"][:], rtol=1e-5
+    )
+    np.testing.assert_allclose(
+        vmec.variables["volavgB"][:], desc.variables["volavgB"][:], rtol=1e-5
+    )
+    np.testing.assert_allclose(
+        vmec.variables["betatotal"][:], desc.variables["betatotal"][:], rtol=1e-5
+    )
+    np.testing.assert_allclose(
+        vmec.variables["betapol"][:], desc.variables["betapol"][:], rtol=1e-5
+    )
+    np.testing.assert_allclose(
+        vmec.variables["betator"][:], desc.variables["betator"][:], rtol=1e-5
+    )
+    np.testing.assert_allclose(
+        vmec.variables["ctor"][:],
+        desc.variables["ctor"][:],
+        atol=1e-9,  # it is a zero current solve
+    )
+    np.testing.assert_allclose(
+        vmec.variables["rbtor"][:], desc.variables["rbtor"][:], rtol=1e-5
+    )
+    np.testing.assert_allclose(
+        vmec.variables["rbtor0"][:], desc.variables["rbtor0"][:], rtol=1e-5
+    )
+    np.testing.assert_allclose(
+        vmec.variables["b0"][:], desc.variables["b0"][:], rtol=4e-3
+    )
+    np.testing.assert_allclose(
+        vmec.variables["buco"][20:100], desc.variables["buco"][20:100], atol=1e-15
+    )
+    np.testing.assert_allclose(
+        vmec.variables["bvco"][20:100], desc.variables["bvco"][20:100], rtol=1e-5
+    )
+    np.testing.assert_allclose(
+        vmec.variables["vp"][20:100], desc.variables["vp"][20:100], rtol=3e-4
+    )
+    np.testing.assert_allclose(
+        vmec.variables["bdotb"][20:100], desc.variables["bdotb"][20:100], rtol=3e-4
+    )
+    np.testing.assert_allclose(
+        vmec.variables["jdotb"][20:100],
+        desc.variables["jdotb"][20:100],
+        atol=4e-3,  # nearly zero bc is vacuum
+    )
+    np.testing.assert_allclose(
+        vmec.variables["jcuru"][20:100], desc.variables["jcuru"][20:100], atol=2
+    )
+    np.testing.assert_allclose(
+        vmec.variables["jcurv"][20:100], desc.variables["jcurv"][20:100], rtol=2
+    )
+    np.testing.assert_allclose(
+        vmec.variables["DShear"][20:100], desc.variables["DShear"][20:100], rtol=3e-2
+    )
+    np.testing.assert_allclose(
+        vmec.variables["DCurr"][20:100],
+        desc.variables["DCurr"][20:100],
+        atol=1e-4,  # nearly zero bc vacuum
+    )
+    np.testing.assert_allclose(
+        vmec.variables["DWell"][20:100], desc.variables["DWell"][20:100], rtol=1e-2
```

Comment:
Why exclude points close to the axis? Because of strong variations?

Reply:
b/c VMEC is notorious for being inaccurate near-axis. But in this case the two solutions (the DESC and VMEC) one are not exactly similar (even visually) near-axis, so we also want to exclude it for that reason.

I dont need this test to be a validation of DESC v VMEC, I just want it to tell me if I am way off in the asymmetric quantities basically